### PR TITLE
Added the following metrics:

### PR DESCRIPTION
**Pull request by David-VTUK**
- rancherLatestMajorVersion
- rancherLatestMinorVersion
- rancherLatestPatchVersion
- managedNodeCount

### CODE REVIEW

File: query/rancher/rancher.go [hunk 1]
```diff
@@ -66,6 +71,44 @@ func (r Client) GetK8sDistributions() (map[string]int, error) {
...
+	defer resp.Body.Close()
+	if err != nil {
+		log.Fatal("failed to open url")
+		return nil, err
+	}
+
+	bodyBytes, err := io.ReadAll(resp.Body)
+
+	if err != nil {
+		log.Fatal("failed to read HTTP body")
```
Comment (ebauman) [inline query/rancher/rancher.go]: I'm not sure fatal is what we want here? Should we be killing off this thing if we can't talk to github?

File: query/rancher/rancher.go [hunk 2]
```diff
@@ -66,6 +71,44 @@ func (r Client) GetK8sDistributions() (map[string]int, error) {
...
+		return nil, err
+	}
+
+	val := gjson.Get(string(bodyBytes), "tag_name")
+
+	result, err := semver.Parse(TrimVersionChar(val.String()))
+
+	if err != nil {
+		log.Fatal("failed to parse Version")
```
Comment (ebauman) [inline query/rancher/rancher.go]: same re: Fatal

**Review by ebauman — CHANGES_REQUESTED** [review]

File: query/rancher/rancher.go (hunk 1 above)
Comment (David-VTUK) [reply query/rancher/rancher.go]: Good point, I'm guessing we should be handling this in collector.go? I'll remove the log.fatal messages from GetLatestRancherVersion() in favour of handling the errors in collector.go

File: query/rancher/rancher.go (hunk 2 above)
Comment (David-VTUK) [reply query/rancher/rancher.go]: As above

**Pull request merged**